po/id.po

```diff
@@ -1156,22 +1158,22 @@ msgstr "DEST"
 
 #: app/flatpak-builtins-create-usb.c:48
 msgid "Allow partial commits in the created repo"
-msgstr ""
+msgstr "Izinkan komit parsial dalam repo yang dibuat"
 
 #: app/flatpak-builtins-create-usb.c:126
-#, fuzzy, c-format
+#, c-format
 msgid "Related ref '%s' is only partially installed"
-msgstr "Ekstensi %s yang diminta hanya terpasang sebagian"
+msgstr "Ref '%s' terkait hanya dipasang sebagian"
 
 #: app/flatpak-builtins-create-usb.c:157
 #, c-format
 msgid "Warning: Omitting related ref ‘%s’ because it is not installed.\n"
 msgstr "Peringatan: Menghilangkan ref terkait '%s' karena tidak dipasang.\n"
 
 #: app/flatpak-builtins-create-usb.c:164
-#, fuzzy, c-format
+#, c-format
 msgid "Warning: Omitting related ref ‘%s’ because it is partially installed.\n"
-msgstr "Peringatan: Menghilangkan ref terkait '%s' karena tidak dipasang.\n"
+msgstr "Peringatan: Menghilangkan ref terkait ‘%s 'karena dipasang sebagian.\n"
```

Comment:
Looks like a typo: whitespace in the wrong place and mismatched apostrophes.

Reply:
Fix on https://github.com/flatpak/flatpak/pull/3629/commits/fea5074840cdd1e635e68d202e3772088327e5d9